app/src/main/java/com/karrar/movieapp/ui/main/StartActivity.kt

```diff
@@ -22,22 +27,20 @@ class StartActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    ActorItem(
+                        modifier = Modifier.padding(8.dp),
+                        actorName = "Test test",
+                        actorImageUrl = "https://image.tmdb.org/t/p/w500/8V1XPEDLtJoxOWlE6gYzn2sQaMp.jpg",
+                        onClick = { }
```

Comment:
i think you are calling the actor item here just for testing if that the case don't forget to remove them before committing your work